src/main/java/org/spongepowered/common/mixin/core/world/MixinWorldServer.java

```diff
@@ -237,7 +233,7 @@
     private static final String PROFILER_ESS = "Lnet/minecraft/profiler/Profiler;endStartSection(Ljava/lang/String;)V";
 
     private static final Vector3i BLOCK_MIN = new Vector3i(-30000000, 0, -30000000);
-    private static final Vector3i BLOCK_MAX = new Vector3i(30000000, 256, 30000000);
+    private static final Vector3i BLOCK_MAX = new Vector3i(30000000, 256, 30000000).sub(Vector3i.ONE);
```

Comment:
You can safely remove these and switch to the getter of the superclass. 

(They are only used inside checkBlockBounds)